src/topic_training/src/Begin_Here.md

```diff
@@ -0,0 +1,122 @@
+# WRoverPlayground - Topic Training
+Now that you have completed the Tools Training, you are ready to learn about ROS topics. From the Setup Training, you should be familiar with:
+- the basics of ROS Nodes
+- rosnode
+- and roscore.
+
+Once you complete this training, you should be familiar with:
+- the purpose of topics
+- the relationship between Publishers and Subscribers
+- setting up Publishers and Subscribers
+- running Publishers and Subscribers
+- rqt_graph
+- rqt_plot
+- custom topic types
+- and rostopic type {topic}.
+
+## The Purpose of Topics
+The word "topic" sounds very general, but it is actually a specific concept that allows ROS nodes to communicate with each other. This is extremely important because one node cannot possibly contain all the information a rover would need.
+
+Topics allow us to specialize nodes, so they can execute their own code/tasks, while also sending and receiving information. For example, a node could receive data and send it to another one, a node could receive and analyze data and have another node do some action with it, a node could receive data from multiple nodes, etc.
+
+## Publisher/Subscriber Relationship
+The Publisher and Subscriber are both nodes that are connected by a topic, allowing them to send messages between themselves. The Publisher node publishes messages on the topic, while the Subscriber node subscribes to the same topic to receive the Publisher's messages. 
```

Comment:
Nit-pick; don't use a topic to send data to yourself (comparatively *very* inefficient). 
```suggestion
The Publisher and Subscriber are both nodes that are connected by a topic, allowing them to send messages between each other. The Publisher node publishes messages on the topic, while the Subscriber node subscribes to the same topic to receive the Publisher's messages. 
```